index.html

```diff
@@ -164,210 +164,257 @@
         "#constraints"></a>). Two new constraints allow an application to observe what type of
         information was acquired.
       </p>
+      <div class="note">
+        <p>
+          Though all constraints are supported for display device capture, not all constraints will
+          have an effect. Usually, this is because an output or display device cannot alter its
+          behavior in a way that the constraint implies, just as a video camera cannot alter
+          <a href="https://w3c.github.io/mediacapture-main/#def-constraint-volume">volume</a>.
```

Comment:
I'm merely being consistent with the existing API, namely that if you specify "volume" on video, you get what you deserve (i.e., overconstrained, I think).